samples/.NET/NUnit/PageTestWithArtifact.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.Playwright;
+using Microsoft.Playwright.NUnit;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+
+namespace PlaywrightTests
+{
+    /*
+     * PageTestWithArtifact is a base class for tests that require Playwright page and context.
+     * It provides a setup and teardown methods for the test to run in a browser context.
+     * It also provides a way to enable trace, screenshot, and video artifacts for the test.
+     * Users need to inherit this class instead of PageTest to write tests.
+     */
+    [TestFixture]
+    public class PageTestWithArtifact : PageTest
```

Comment:
Can we provide the content of setup and teardown methods in separate function for all the examples and just have a single line to call these methods in the setup and teardown file? 

I feel it would be easier for customers to copy these separate methods in their existing fixtures and just modify the existing setup and teardown methods to call the separate methods they had copied earlier.

Reply:
I wanted to keep this as simple boilerplate code more as guidance, because users would have thier own setup/teardown and they should mostly take the required code carefully. I have already kept service specific code like createBrowser in a function.